examples/visualization/styles/helpers/color-categories-style.html

```diff
@@ -45,26 +45,7 @@
         }
       });
 
-      const roomTypeStyle = carto.viz.colorCategoriesStyle('room_type', {
-        // categories: ['Entire home/apt', 'Private room', 'Shared room'],
-
-        // categoryColors accepts an array of HEX color strings,
-        // or a string pointing to a CARTOColors palette
-        // palette: ['#88ccee', '#cc6677', '#ddcc77'],
-
-        // HEXA color for null values
-        nullColor: '#00000010',
-
-        // HEXA color for categories not included in
-        // categories array
-        othersColor: '#FFFFFF10'
-      });
-
-      // roomTypeStyle.getRadius = 100;
-      
-      roomTypeStyle
-
-
+      const roomTypeStyle = carto.viz.colorCategoriesStyle('room_type');
```

Comment:
same, I would try to display the defaults been applied, so it's easier for the user to 'play' with them